### PR TITLE
Update applications-databases-postgresql

### PR DESCRIPTION
**Pull request by omercier**
## Target version (i.e. version that this PR changes)

- [ ] 21.10.x
- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-postgresql.md [hunk 1]
```diff
@@ -2,148 +2,170 @@
...
-| Metric Name                                        | Unit  |
-|:---------------------------------------------------|:------|
-| *db_name~table_name*#table.space.usage.bytes       | B     |
-| *db_name~table_name*#table.space.free.bytes        | B     |
-| *db_name~table_name*#table.dead_tuple.bytes        | B     |
-| *db_name~index_name*#index.space.usage.bytes       | B     |
-| *db_name~index_name*#index.leaf_density.percentage | %     |
+| Métrique                                             | Unité |
+|:-----------------------------------------------------|:------|
+| *databases*~*tables*#table.space.usage.bytes         | B     |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-postgresql.md:72]: db_name et table_name c'est plus clair

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-postgresql.md [hunk 2]
```diff
@@ -166,100 +188,517 @@ GRANT EXECUTE ON FUNCTION pgstattuple(regclass) TO centreonro;
...
+3. Appliquez le modèle d'hôte **App-DB-Postgres-custom**. Une liste de macros apparaît. Les macros vous permettent de définir comment le connecteur se connectera à la ressource, ainsi que de personnaliser le comportement du connecteur.
+4. Renseignez les macros désirées. Attention, certaines macros sont obligatoires.
+
+| Macro                | Description                                                                                           | Valeur par défaut | Obligatoire |
+|:---------------------|:------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| POSTGRESUSERNAME     | User name used to connect to the database                                                             |                   |      X      |
+| POSTGRESPASSWORD     | Password for the defined user name                                                                    |                   |      X      |
+| POSTGRESPORT         | Database Server Port                                                                                  | 5432              |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-postgresql.md:290]:
```suggestion
| POSTGRESPORT         | Database server port                                                                                  | 5432              |             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-postgresql.md [hunk 3]
```diff
@@ -166,100 +188,517 @@ GRANT EXECUTE ON FUNCTION pgstattuple(regclass) TO centreonro;
...
+| POSTGRESDATABASE     | Database Name                                                                                         | postgres          |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-postgresql.md:291]:
```suggestion
| POSTGRESDATABASE     | Database name                                                                                         | postgres          |             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-postgresql.md [hunk 4]
```diff
@@ -166,100 +188,517 @@ GRANT EXECUTE ON FUNCTION pgstattuple(regclass) TO centreonro;
...
-| X           | POSTGRESPASSWORD     |                                                                                   |
-|             | POSTGRESPORT         | Port used (Default: 5432)                                                         |
-|             | POSTGRESDATABASE     | Database connection (Default: postgres)                                           |
-|             | POSTGRESEXTRAOPTIONS | Any extra option you may want to add to every command line (eg. a --verbose flag) |
+</TabItem>
+<TabItem value="Cache-Hitratio" label="Cache-Hitratio">
+
+| Macro        | Description                                                                                         | Valeur par défaut                     | Obligatoire |
+|:-------------|:----------------------------------------------------------------------------------------------------|:--------------------------------------|:-----------:|
+| FILTER       | Filter databases                                                                                    | ^(?!(postgres\|template1\|template0)) |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-postgresql.md:326]: filter what?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-postgresql.md [hunk 5]
```diff
@@ -166,100 +188,517 @@ GRANT EXECUTE ON FUNCTION pgstattuple(regclass) TO centreonro;
...
+| WARNING      | Warning threshold in milliseconds                                                                   |                   |             |
+| CRITICAL     | Critical threshold in milliseconds                                                                  |                   |             |
+| EXTRAOPTIONS | Any extra option you may want to add to the command (E.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles) |                   |             |
+
+</TabItem>
+<TabItem value="Connection-Number" label="Connection-Number">
+
+| Macro        | Description                                                                                         | Valeur par défaut                     | Obligatoire |
+|:-------------|:----------------------------------------------------------------------------------------------------|:--------------------------------------|:-----------:|
+| FILTER       | Filter databases                                                                                    | ^(?!(postgres\|template1\|template0)) |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-postgresql.md:345]: filter what?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-postgresql.md [hunk 6]
```diff
@@ -166,100 +188,517 @@ GRANT EXECUTE ON FUNCTION pgstattuple(regclass) TO centreonro;
...
+| CRITICAL     | Critical threshold in percent                                                                       | 95                                    |             |
+| WARNING      | Warning threshold in percent                                                                        | 90                                    |             |
+| EXTRAOPTIONS | Any extra option you may want to add to the command (E.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles) | --verbose                             |             |
+
+</TabItem>
+<TabItem value="Database-Size" label="Database-Size">
+
+| Macro          | Description                                                                                         | Valeur par défaut | Obligatoire |
+|:---------------|:----------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| FILTERDATABASE | Filter database to checks (Can use regexp)                                                          | .*                |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-postgresql.md:355]:
```suggestion
| FILTERDATABASE | Filter database to checks (can use regexp)                                                          | .*                |             |
```

Comment (cg-tw) [reply i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-postgresql.md:355]: ?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-postgresql.md [hunk 7]
```diff
@@ -166,100 +188,517 @@ GRANT EXECUTE ON FUNCTION pgstattuple(regclass) TO centreonro;
...
+| WARNINGSIZE    | Warning threshold in bytes, maximum size allowed                                                    |                   |             |
+| CRITICALSIZE   | Critical threshold in bytes, maximum size allowed                                                   |                   |             |
+| EXTRAOPTIONS   | Any extra option you may want to add to the command (E.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles) |                   |             |
+
+</TabItem>
+<TabItem value="Locks" label="Locks">
+
+| Macro        | Description                                                                                                                       | Valeur par défaut                     | Obligatoire |
+|:-------------|:----------------------------------------------------------------------------------------------------------------------------------|:--------------------------------------|:-----------:|
+| FILTER       | Filter databases                                                                                                                  | ^(?!(postgres\|template1\|template0)) |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-postgresql.md:365]: filter what?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-postgresql.md [hunk 8]
```diff
@@ -166,100 +188,517 @@ GRANT EXECUTE ON FUNCTION pgstattuple(regclass) TO centreonro;
...
+| WARNING      | Warning threshold. (example: "total=250,waiting=5,exclusive=20") 'total', 'waiting', or the name of a lock type used by Postgres  | total=200                             |             |
+| EXTRAOPTIONS | Any extra option you may want to add to the command (E.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles)                               | --verbose                             |             |
+
+</TabItem>
+<TabItem value="Query-Time" label="Query-Time">
 
-## Comment puis-je tester le Plugin et que signifient les options des commandes ?
+| Macro        | Description                                                                                         | Valeur par défaut                     | Obligatoire |
+|:-------------|:----------------------------------------------------------------------------------------------------|:--------------------------------------|:-----------:|
+| FILTER       | Filter databases                                                                                    | ^(?!(postgres\|template1\|template0)) |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-postgresql.md:375]: filter what?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-postgresql.md [hunk 9]
```diff
@@ -166,100 +188,517 @@ GRANT EXECUTE ON FUNCTION pgstattuple(regclass) TO centreonro;
...
+| WARNING      | Thresholds.                                                                                         |                   |             |
+| CRITICAL     |  Thresholds.                                                                                        |                   |             |
+| EXTRAOPTIONS | Any extra option you may want to add to the command (E.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles) |                   |             |
+
+</TabItem>
+<TabItem value="Statistics" label="Statistics">
+
+| Macro               | Description                                                                                         | Valeur par défaut                      | Obligatoire |
+|:--------------------|:----------------------------------------------------------------------------------------------------|:---------------------------------------|:-----------:|
+| FILTER              | Filter database (can be a regexp)                                                                   | ^(?!(postgres\|template1\|template0)$) |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-postgresql.md:396]: filter what?

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-postgresql.md [hunk 10]
```diff
@@ -2,148 +2,170 @@
...
-| *db_name~table_name*#table.dead_tuple.bytes        | B     |
-| *db_name~index_name*#index.space.usage.bytes       | B     |
-| *db_name~index_name*#index.leaf_density.percentage | %     |
+| Métrique                                             | Unité |
+|:-----------------------------------------------------|:------|
+| *databases*~*tables*#table.space.usage.bytes         | B     |
+| *databases*~*tables*#table.space.free.bytes          | B     |
+| *databases*~*tables*#table.dead\_tuple.bytes         | B     |
+| *databases*~*indexes*#index.space.usage.bytes        | B     |
+| *databases*~*indexes*#index.leaf\_density.percentage | %     |
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-postgresql.md:76]:
```suggestion
| *db_name~table_name*#table.space.usage.bytes         | B     |
| *db_name~table_name*#table.space.free.bytes          | B     |
| *db_name~table_name*#table.dead\_tuple.bytes         | B     |
| *db_name~index_name*#index.space.usage.bytes        | B     |
| *db_name~index_name*#index.leaf\_density.percentage | %     |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-postgresql.md (hunk 4 above)
Comment (omercier) [reply i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-postgresql.md:326]: database names ?

File: pp/integrations/plugin-packs/procedures/applications-databases-postgresql.md [hunk 11]
```diff
@@ -2,141 +2,163 @@
...
-| *db_name~table_name*#table.dead_tuple.bytes        | B     |
-| *db_name~index_name*#index.space.usage.bytes       | B     |
-| *db_name~index_name*#index.leaf_density.percentage | %     |
+| Metric name                                         | Unit  |
+|:----------------------------------------------------|:------|
+| *databases*~*tables*#table.space.usage.bytes        | B     |
+| *databases*~*tables*#table.space.free.bytes         | B     |
+| *databases*~*tables*#table.dead_tuple.bytes         | B     |
+| *databases*~*indexes*#index.space.usage.bytes       | B     |
+| *databases*~*indexes*#index.leaf_density.percentage | %     |
```
Comment (omercier) [inline pp/integrations/plugin-packs/procedures/applications-databases-postgresql.md:75]:
```suggestion
| *db_name~table_name*#table.space.usage.bytes       | B     |
| *db_name~table_name*#table.space.free.bytes        | B     |
| *db_name~table_name*#table.dead_tuple.bytes        | B     |
| *db_name~index_name*#index.space.usage.bytes       | B     |
| *db_name~index_name*#index.leaf_density.percentage | %     |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-postgresql.md [hunk 12]
```diff
@@ -166,100 +188,517 @@ GRANT EXECUTE ON FUNCTION pgstattuple(regclass) TO centreonro;
...
-| X           | POSTGRESPASSWORD     |                                                                                   |
-|             | POSTGRESPORT         | Port used (Default: 5432)                                                         |
-|             | POSTGRESDATABASE     | Database connection (Default: postgres)                                           |
-|             | POSTGRESEXTRAOPTIONS | Any extra option you may want to add to every command line (eg. a --verbose flag) |
+</TabItem>
+<TabItem value="Cache-Hitratio" label="Cache-Hitratio">
+
+| Macro        | Description                                                                                         | Valeur par défaut                     | Obligatoire |
+|:-------------|:----------------------------------------------------------------------------------------------------|:--------------------------------------|:-----------:|
+| FILTER       | Filter databases                                                                                    | ^(?!(postgres\|template1\|template0)) |             |
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-postgresql.md:326]:
```suggestion
| FILTER       | Filter databases by name (can be a regexp).                                                                                    | ^(?!(postgres\|template1\|template0)) |             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-postgresql.md [hunk 13]
```diff
@@ -166,100 +188,517 @@ GRANT EXECUTE ON FUNCTION pgstattuple(regclass) TO centreonro;
...
+| WARNING      | Warning threshold in milliseconds                                                                   |                   |             |
+| CRITICAL     | Critical threshold in milliseconds                                                                  |                   |             |
+| EXTRAOPTIONS | Any extra option you may want to add to the command (E.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles) |                   |             |
+
+</TabItem>
+<TabItem value="Connection-Number" label="Connection-Number">
+
+| Macro        | Description                                                                                         | Valeur par défaut                     | Obligatoire |
+|:-------------|:----------------------------------------------------------------------------------------------------|:--------------------------------------|:-----------:|
+| FILTER       | Filter databases                                                                                    | ^(?!(postgres\|template1\|template0)) |             |
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-postgresql.md:345]:
```suggestion
| FILTER       | Filter databases by name (can be a regexp).                                                                                    | ^(?!(postgres\|template1\|template0)) |             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-postgresql.md [hunk 14]
```diff
@@ -166,100 +188,517 @@ GRANT EXECUTE ON FUNCTION pgstattuple(regclass) TO centreonro;
...
+| WARNINGSIZE    | Warning threshold in bytes, maximum size allowed                                                    |                   |             |
+| CRITICALSIZE   | Critical threshold in bytes, maximum size allowed                                                   |                   |             |
+| EXTRAOPTIONS   | Any extra option you may want to add to the command (E.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles) |                   |             |
+
+</TabItem>
+<TabItem value="Locks" label="Locks">
+
+| Macro        | Description                                                                                                                       | Valeur par défaut                     | Obligatoire |
+|:-------------|:----------------------------------------------------------------------------------------------------------------------------------|:--------------------------------------|:-----------:|
+| FILTER       | Filter databases                                                                                                                  | ^(?!(postgres\|template1\|template0)) |             |
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-postgresql.md:365]:
```suggestion
| FILTER       | Filter databases by name (can be a regexp).                                                                                                                  | ^(?!(postgres\|template1\|template0)) |             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-postgresql.md [hunk 15]
```diff
@@ -166,100 +188,517 @@ GRANT EXECUTE ON FUNCTION pgstattuple(regclass) TO centreonro;
...
+| WARNING      | Warning threshold. (example: "total=250,waiting=5,exclusive=20") 'total', 'waiting', or the name of a lock type used by Postgres  | total=200                             |             |
+| EXTRAOPTIONS | Any extra option you may want to add to the command (E.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles)                               | --verbose                             |             |
+
+</TabItem>
+<TabItem value="Query-Time" label="Query-Time">
 
-## Comment puis-je tester le Plugin et que signifient les options des commandes ?
+| Macro        | Description                                                                                         | Valeur par défaut                     | Obligatoire |
+|:-------------|:----------------------------------------------------------------------------------------------------|:--------------------------------------|:-----------:|
+| FILTER       | Filter databases                                                                                    | ^(?!(postgres\|template1\|template0)) |             |
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-postgresql.md:375]:
```suggestion
| FILTER       | Filter databases by name (can be a regexp).                                                                                    | ^(?!(postgres\|template1\|template0)) |             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-postgresql.md [hunk 16]
```diff
@@ -166,100 +188,517 @@ GRANT EXECUTE ON FUNCTION pgstattuple(regclass) TO centreonro;
...
+| WARNING      | Thresholds.                                                                                         |                   |             |
+| CRITICAL     |  Thresholds.                                                                                        |                   |             |
+| EXTRAOPTIONS | Any extra option you may want to add to the command (E.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles) |                   |             |
+
+</TabItem>
+<TabItem value="Statistics" label="Statistics">
+
+| Macro               | Description                                                                                         | Valeur par défaut                      | Obligatoire |
+|:--------------------|:----------------------------------------------------------------------------------------------------|:---------------------------------------|:-----------:|
+| FILTER              | Filter database (can be a regexp)                                                                   | ^(?!(postgres\|template1\|template0)$) |             |
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-postgresql.md:396]:
```suggestion
| FILTER              | Filter databases by name (can be a regexp).                                                                   | ^(?!(postgres\|template1\|template0)$) |             |
```

File: pp/integrations/plugin-packs/procedures/applications-databases-postgresql.md [hunk 17]
```diff
@@ -165,100 +187,517 @@ GRANT EXECUTE ON FUNCTION pgstattuple(regclass) TO centreonro;
...
+| WARNING      | Thresholds.                                                                                         |                   |             |
+| CRITICAL     |  Thresholds.                                                                                        |                   |             |
+| EXTRAOPTIONS | Any extra option you may want to add to the command (E.g. a --verbose flag). All options are listed [here](#available-options) |                   |             |
+
+</TabItem>
+<TabItem value="Statistics" label="Statistics">
+
+| Macro               | Description                                                                                         | Default value                          | Mandatory   |
+|:--------------------|:----------------------------------------------------------------------------------------------------|:---------------------------------------|:-----------:|
+| FILTER              | Filter database (can be a regexp)                                                                   | ^(?!(postgres\|template1\|template0)$) |             |
```
Comment (omercier) [inline pp/integrations/plugin-packs/procedures/applications-databases-postgresql.md:397]:
```suggestion
| FILTER              | Filter databases by name (can be a regexp).                                                                  | ^(?!(postgres\|template1\|template0)$) |             |
```

File: pp/integrations/plugin-packs/procedures/applications-databases-postgresql.md [hunk 18]
```diff
@@ -165,100 +187,517 @@ GRANT EXECUTE ON FUNCTION pgstattuple(regclass) TO centreonro;
...
+| WARNING      | Warning threshold. (example: "total=250,waiting=5,exclusive=20") 'total', 'waiting', or the name of a lock type used by Postgres  | total=200                             |             |
+| EXTRAOPTIONS | Any extra option you may want to add to the command (E.g. a --verbose flag). All options are listed [here](#available-options)                               | --verbose                             |             |
+
+</TabItem>
+<TabItem value="Query-Time" label="Query-Time">
+
+| Macro        | Description                                                                                         | Default value                         | Mandatory   |
+|:-------------|:----------------------------------------------------------------------------------------------------|:--------------------------------------|:-----------:|
+| FILTER       | Filter databases                                                                                    | ^(?!(postgres\|template1\|template0)) |             |
```
Comment (omercier) [inline pp/integrations/plugin-packs/procedures/applications-databases-postgresql.md:376]:
```suggestion
| FILTER       | Filter databases by name (can be a regexp).                                                                                    | ^(?!(postgres\|template1\|template0)) |             |
```

File: pp/integrations/plugin-packs/procedures/applications-databases-postgresql.md [hunk 19]
```diff
@@ -165,100 +187,517 @@ GRANT EXECUTE ON FUNCTION pgstattuple(regclass) TO centreonro;
...
+| WARNINGSIZE    | Warning threshold in bytes, maximum size allowed                                                    |                   |             |
+| CRITICALSIZE   | Critical threshold in bytes, maximum size allowed                                                   |                   |             |
+| EXTRAOPTIONS   | Any extra option you may want to add to the command (E.g. a --verbose flag). All options are listed [here](#available-options) |                   |             |
+
+</TabItem>
+<TabItem value="Locks" label="Locks">
+
+| Macro        | Description                                                                                                                       | Default value                         | Mandatory   |
+|:-------------|:----------------------------------------------------------------------------------------------------------------------------------|:--------------------------------------|:-----------:|
+| FILTER       | Filter databases                                                                                                                  | ^(?!(postgres\|template1\|template0)) |             |
```
Comment (omercier) [inline pp/integrations/plugin-packs/procedures/applications-databases-postgresql.md:366]:
```suggestion
| FILTER       | Filter databases by name (can be a regexp).                                                                                                                  | ^(?!(postgres\|template1\|template0)) |             |
```

File: pp/integrations/plugin-packs/procedures/applications-databases-postgresql.md [hunk 20]
```diff
@@ -165,100 +187,517 @@ GRANT EXECUTE ON FUNCTION pgstattuple(regclass) TO centreonro;
...
+| WARNING      | Warning threshold in milliseconds                                                                   |                   |             |
+| CRITICAL     | Critical threshold in milliseconds                                                                  |                   |             |
+| EXTRAOPTIONS | Any extra option you may want to add to the command (E.g. a --verbose flag). All options are listed [here](#available-options) |                   |             |
+
+</TabItem>
+<TabItem value="Connection-Number" label="Connection-Number">
+
+| Macro        | Description                                                                                         | Default value                         | Mandatory   |
+|:-------------|:----------------------------------------------------------------------------------------------------|:--------------------------------------|:-----------:|
+| FILTER       | Filter databases                                                                                    | ^(?!(postgres\|template1\|template0)) |             |
```
Comment (omercier) [inline pp/integrations/plugin-packs/procedures/applications-databases-postgresql.md:346]:
```suggestion
| FILTER       | Filter databases by name (can be a regexp).                                                                                    | ^(?!(postgres\|template1\|template0)) |             |
```

File: pp/integrations/plugin-packs/procedures/applications-databases-postgresql.md [hunk 21]
```diff
@@ -165,100 +187,517 @@ GRANT EXECUTE ON FUNCTION pgstattuple(regclass) TO centreonro;
...
-| X           | POSTGRESUSERNAME     |                                                                                   |
-| X           | POSTGRESPASSWORD     |                                                                                   |
-|             | POSTGRESPORT         | Port used (Default: 5432)                                                         |
-|             | POSTGRESDATABASE     | Database connection (Default: postgres)                                           |
-|             | POSTGRESEXTRAOPTIONS | Any extra option you may want to add to every command line (eg. a --verbose flag) |
+| Macro        | Description                                                                                         | Default value                         | Mandatory   |
+|:-------------|:----------------------------------------------------------------------------------------------------|:--------------------------------------|:-----------:|
+| FILTER       | Filter databases                                                                                    | ^(?!(postgres\|template1\|template0)) |             |
```
Comment (omercier) [inline pp/integrations/plugin-packs/procedures/applications-databases-postgresql.md:327]:
```suggestion
| FILTER       | Filter databases by name (can be a regexp).                                                                                    | ^(?!(postgres\|template1\|template0)) |             |
```

**Review by cg-tw — APPROVED** [review]

**Pull request merged**